Release 0 13 0 (#633)
kestra-io/docs: content/docs/05.developer-guide/06.storage.md

  to: "/upload/file.ion"
```

> If you need to access data from the internal storage in a task that was not designed to handle an internal storage URI, you can use the Pebble [read](./03.variables/04.function/read.md) to read the file content as a string.

Reviewer: ```suggestion
If you need to access data from the internal storage, you can use the Pebble [read](./03.variables/04.function/read.md) function to read the file's content as a string.
```

not designed to do X could be interpreted as a discouraged pattern
